dstep/driver/Application.d

```diff
@@ -48,16 +46,22 @@ class Application : DStack.Application
 
     protected override void run ()
     {
+        derr.writefln(arguments.rawArgs);
```

Comment:
Moreover, I do not see the output from this neither on my machine nor on the build bot Am I supposed to print to ```stderr``` in some other way?